### ISSUE TITLE
New tool Addition: Panaroo 

### ISSUE THREAD

**Issue by SaimMomin12** [galaxyproject/tools-iuc#6581]
One of our bachelor students  at the University of Freiburg (as a part of his Bachelor Project), is currently working on integrating the Panaroo tool into Galaxy. While a pull request (PR) has not yet been created, we will submit an official PR once the tool is ready for review. Our target is to have the tool available by early January next year.

**Comment by wm75** [comment]
For reference:

bioconda recipe: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/panaroo/meta.yaml
original publication: https://doi.org/10.1186/s13059-020-02090-4
publication comparing pangenome tools like Roary, Panaroo and PanTa: https://doi.org/10.1186/s13059-024-03362-z

**Comment by wm75** [comment]
PanTa - https://github.com/amromics/panta - might be another interesting tool by itself since it seems to be even more efficient than Panaroo.

**Comment by wm75** [comment]
Interesting: there is https://github.com/usegalaxy-au/tools-au/tree/master/tools/panaroo
Why is this AU-specific?

**Comment by mthang** [comment]
@wm75 @SaimMomin12 I am a member of Galaxy Australia and wrapped panaroo. This request came from our research community. You can test it on https://usegalaxy.org.au/ if you want. Any feedback is welcome and appreciated.

**Comment by wm75** [comment]
Thanks @mthang, will definitely give it a try!

**Comment by lsfr3** [comment]
@mthang I am currently trying to use Panaroo (Galaxy Australia) with Prokka outputs generated in the same server, but the Panaroo is not recognizing these outputs as they are .gff3 but apparently the acceptable format is only the version 1 of .gff. Do you know how to overcome this situation? Thank you

**Comment by mthang** [comment]
@lsfr3 I will need to have a second look at the panaroo [documentation](https://gthlab.au/panaroo/#/gettingstarted/quickstart) before adding gff3 file type to the panaroo. It seems like panaroo treats gff and gff3 differently.

**Comment by SaimMomin12** [comment]
New PR ==> https://github.com/galaxyproject/tools-iuc/pull/6811
